src/command_modules/azure-cli-cognitiveservices/setup.py

```diff
@@ -0,0 +1,57 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+    cmdclass = {}
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+]
+
+DEPENDENCIES = [
+    'azure-mgmt-cognitiveservices==1.0.0',
+    'azure-cli-core',
+]
+
+with open('README.rst', 'r', encoding='utf-8') as f:
+    README = f.read()
+
+setup(
+    name='azure-cli-cognitiveservices',
+    version='0.1.0',
+    description='Microsoft Azure Command-Line Tools Cognitive Services Command Module',
+    long_description=README,
```

Comment:
This does not include the HISTORY.rst. See the other setup.py files for examples.